src/processes/follow-topic-process.js

```diff
@@ -134,18 +134,30 @@ const followTopicProcess = async (job, done) => {
 
         await chat.create();
         const channelState = await chat.watch();
-
         if (channelState.messages.length === 0) {
           await chat.sendMessage({
             text: communityMessageFormat.message,
-            user_id: communityMessageFormat.user_id
+            user_id: communityMessageFormat.user_id,
+            is_auto_message: true
           });
 
           try {
             // await chat.stopWatching();
           } catch (error) {
             console.log('::: Error on stopWatching :::', error);
           }
+        } else if (
+          channelState.messages.length === 1 &&
+          channelState.messages[0]?.is_auto_message
+        ) {
+          // add new condition based on ticket ping-4227
+          // check if first message is auto message from sender
+          await chat.sendMessage({
+            text: communityMessageFormat.message,
+            user_id: communityMessageFormat.user_id
+          });
+        } else {
+          console.log(':::  Channel not eligible to receive auto message :::');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Using `console.log` for logging errors or important information is not recommended in production code. Consider using a proper logging library that provides different levels of logging (info, warn, error, etc.) and can write logs to various outputs. This will help with debugging and monitoring your application.
```diff
- console.log(':::  Channel not eligible to receive auto message :::');
+ console.error(':::  Channel not eligible to receive auto message :::');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->